mesa/space.py

```diff
@@ -470,7 +486,11 @@ def move_to_empty(
                 ),
                 DeprecationWarning,
             )
-        num_empty_cells = len(self.empties)
+        if not self._empties:
+            self.build_empties()
+
+        num_empty_cells = len(self._empties)
```

Comment:
Why not do this in 1 line, `num_empty_cells = len(self.empties)`?

Reply:
sounds good, done